3-cloud-computing/3.2.3-auto-scaling.md

```diff
@@ -62,30 +61,37 @@ First, we need to create a **Target Group** for our load balancer to be directed
 1. Open the Amazon [console](https://console.aws.amazon.com).
 2. Select *EC2* from the *Services* menu and click *Target Groups* in the left navigation bar.
 3. Click *Create target group*.
-4. Select *Instances* as the target type, set the name as <your-name>-dob-petclinic, protocol as *TCP* and port *8080*.
-5. Set the *Health check protocol* to *HTTP* with a *Health check path* to */petclinic*.
+4. Select *Instances* as the `target type`, set the name as `<your-name>-dob-petclinic`, protocol as *TCP* and port *8080*.
+5. Set the *Health check protocol* to *HTTP* with a *Health check path* to */petclinic*. (Or any path that will return a 200 status code)
 6. Use Liatrio's recommended tag structure (outlined in 3.1.1) to create tags. Hit *Next*.
-7. Select the target instance, then click *Create target group*.
+7. Register the target instance, then click *Create target group*. Make sure you click "Include as pending below" before creating the Target Group.
 
 Now we can create our Load Balancer.
 
 1. Open the Amazon [console](https://console.aws.amazon.com).
 2. Select *EC2* from the *Services* menu and click *Load Balancers* in the left navigation bar.
 3. Click *Create Load Balancer*.
 4. Under *Network Load Balancer* click *Create*.
-5. Specify a name, such as <your-name>-dob-petclinic, set Scheme to *Internet-facing*, and the *IP address type* as IPv4. 
-6. Leave VPC alone and click two zones for mapping (us-west-1a and us-west-1b)
+5. Specify a name, such as `<your-name>-dob-petclinic`, set Scheme to *Internet-facing*, and the *IP address type* as IPv4. 
+6. Leave VPC alone and click two zones for mapping (us-west-1a and us-west-1b), make note of which zones you choose.
 7. For the *Listener*, set the Protocol to *TCP* and port to *8080*. Then select the recently created target group.
 8. Use Liatrio's recommended tag structure to create tags. Then click *Create load balancer*.
 
+Make sure you can access your website via the Load Balancer DNS name.
+
+**Tips**
+- If you are not able to reach your website check the Target Group you created and ensure your hosts are 'Healthy'
+- If they are not healthy double check your Inbound rules on your security group.
+
 ### Deeper Understanding
 
-- In one of your instances try stopping Tomcat.
+- In one of your instances try stopping your web service.
   - What is the health of the load balancer?
   - How long does it take for the health state to change?
   - Can you still access the application through the load balancer?
 - Change some of the load balancer health check tuning settings and then restart your instance.
   - How long does it take for the health state to change?
+- Modify the load balancer to serve your website on port 80.
```

Comment:
https > http? I would imagine one client one would never serve on http port b/c it encourages breaches.

Reply:
Good point. Given these VMs are isolated in test VPC and we configure them with SSH access only the likely hood of one of these VMs being an entry point for the account as a whole seems small. But this could be updated to say. "Modify the load balancer to server your website on a different port". That was the 'extra' exercise I was interested in them understanding.